Makefile

```diff
@@ -48,6 +48,7 @@ features-check:
 	$(Change_Work_Path) && cargo build --features molc,tokio-runtime,generic-timer --no-default-features
 	$(Change_Work_Path) && cargo build --features molc,async-runtime,generic-timer --no-default-features
 	$(Change_Work_Path) && cargo build --features molc,async-runtime,async-timer --no-default-features
+	rustup target add wasm32-unknown-unknown
```

Comment:
I suggest put this command in CI config and just add a comment here to let user know wasm32-unknown-unknown is a required dependency.

Reply:
done